### ISSUE TITLE
constans crashing at phase 6 in the newest patch 1.7.2

### ISSUE THREAD

**Issue by Myth32733** [Haggle1996/RevolutionPack#649]
[crash-2017-05-28_17.16.14-client.txt](https://github.com/Haggle1996/RevolutionPack/files/1034560/crash-2017-05-28_17.16.14-client.txt)

**Comment by Haggle1996** [comment]
Try 3.7.2

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by DragGon7601** [comment]
This was first brought up in SentinalH's Discord. I asked to see the crash log. He said it was 3.7.2

Looks like Reika added some new stuff to the game and its defaulting to used ID's... Im trying to help now. Will post again If I get his instance working.

**Comment by Haggle1996** [comment]
This problem existed in 3.7.1. 3.7.2 was supposed to fix it.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by Myth32733** [comment]
we sorted it out, by changing the Lumen Regen to id 135

**Comment by DragGon7601** [comment]
My server instance was 3.6.5. So I loaded it up and did a NEI Data dump on Potion IDs. The list if full from 1-126. Then there is Radiation Sickness at 140. The list goes as high as 355 (I did a dump all). So ID other than 140 that is 127 or higher should be fine for new IDs.  In Rev 3.6.5 Potion ID 65 was	"wg.potionKnockbackRes witchinggadgets.common.magic.WGPotion". 

I have now updated to the latest Rev 3. The chroma config I got has this at the end:

```
"potion ids" {
    I:"Growth Hormone ID"=36
    S:"Lumarhea ID"=64
    S:"Lumen Regen ID"=65
    I:"Regen ID"=39
    I:"Saturation ID"=38
    S:"Void Gaze ID"=41
}
```

The witching gadgits config has this:
`I:"Potion ID: Knockback Resistance"=65`

This looks like its going to conflict. I got the same crash and error when I tried to load it. First I tried putting a 2 in front of Lumen Regens ID so you get...

`S:"Lumen Regen ID"=265`

But it complained that it was too large. Thinking that maybe it stopped at 256 I tried 135... 

`S:"Lumen Regen ID"=135`

Yay I got the Main menu... And the world loads. This also worked for Myth32733.
